paasta_tools/cli/utils.py

```diff
@@ -517,25 +517,29 @@ def execute_paasta_serviceinit_on_remote_master(subcommand, cluster, service, in
         return run_paasta_serviceinit(subcommand, master, service, instances, cluster, stream, **kwargs)
 
 
-def run_paasta_metastatus(master, humanize, groupings, verbose=0):
+def run_paasta_metastatus(master, humanize, groupings, verbose=0, autoscaling_info=False):
     if verbose > 0:
         verbose_flag = "-%s" % ('v' * verbose)
         timeout = 120
     else:
         verbose_flag = ''
         timeout = 20
+    autoscaling_flag = "-a" if autoscaling_info else ""
+    if autoscaling_flag and verbose < 2:
+        verbose_flag = '-vv'
     humanize_flag = "-H" if humanize else ''
     groupings_flag = "-g %s" % " ".join(groupings) if groupings else ''
-    cmd_args = " ".join(filter(None, [verbose_flag, humanize_flag, groupings_flag]))
-    command = ('ssh -A -n -o StrictHostKeyChecking=no %s sudo paasta_metastatus %s' % (
+    cmd_args = " ".join(filter(None, [verbose_flag, humanize_flag, groupings_flag, autoscaling_flag]))
+    command = ('ssh -A -n -o StrictHostKeyChecking=no %s paasta_metastatus %s' % (
```

Comment:
I thought so too. But it works when I run `/usr/bin/paasta_metastatus -vvv` without the sudo on a master so I guess it only needs credentials to make updates to mesos?